providers/softlayer/src/main/java/org/jclouds/softlayer/features/LoadBalancerApi.java

```diff
@@ -0,0 +1,353 @@
+    */
+   @GET
+   @Named("LoadBalancer:getStatistics")
+   @Path("SoftLayer_Network_LBaaS_LoadBalancer/getLoadBalancerStatistics/{uuid}")
+   @Produces(MediaType.APPLICATION_JSON)
+   @Fallback(NullOnNotFoundOr404.class)
+   LoadBalancerStatistics getLoadBalancerStatistics(@PathParam("uuid") String uuid);
+
+   /**
+    * Returns load balancers members
+    * @param id id of the targeted load balancer
+    * @reutrn members of the load balancer
+    * @see <a href="https://sldn.softlayer.com/reference/services/softlayer_network_lbaas_loadbalancer/getMembers" />
+    */
+   @GET
+   @Named("LoadBalancer:getMembers")
+   @Path("SoftLayer_Network_LBaaS_LoadBalancer/{id}/getMembers")
+   @Produces(MediaType.APPLICATION_JSON)
+   @Fallback(Fallbacks.EmptyListOnNotFoundOr404.class)
+   List<LoadBalancerMember> getLoadBalancerMembers(@PathParam("id") long id);
+
+   /**
+    * Returns load balancers members healths
+    * @param uuid uuid of the targeted load balancer
+    * @reutrn members healths of the load balancer
+    * @see <a href="https://sldn.softlayer.com/reference/services/softlayer_network_lbaas_loadbalancer/getLoadBalancerMemberHealth" />
+    */
+   @GET
+   @Named("LoadBalancer:getLoadBalancerMemberHealth")
+   @Path("SoftLayer_Network_LBaaS_LoadBalancer/getLoadBalancerMemberHealth/{uuid}")
+   @Produces(MediaType.APPLICATION_JSON)
+   @Fallback(Fallbacks.EmptyListOnNotFoundOr404.class)
+   List<LoadBalancerMemberPoolHealth> getLoadBalancerMemberHealth(@PathParam("uuid") String uuid);
+
+   /**
+    * Returns load balancers member with the given id
+    * @param id id of the targeted load balancer member
+    * @reutrn member the load balancer
+    * @see <a href="https://sldn.softlayer.com/reference/services/softlayer_network_lbaas_member/getobject" />
+    */
+   @GET
+   @Named("LoadBalancerMember:getLoadBalancerMember")
+   @Path("SoftLayer_Network_LBaaS_Member/{id}/getObject")
+   @Produces(MediaType.APPLICATION_JSON)
+   @Fallback(NullOnNotFoundOr404.class)
+   LoadBalancerMember getLoadBalancerMember(@PathParam("id") int id);
+
+   /**
+    * Add load balancer members
+    * @param LoadBalancerMembersToBeAdded the load balancer member creation definition of LoadBalancerMember.AddLoadBalancerMember
+    *                                     along with the uuid of the targeted load balancer
+    * @reutrn new load balancer object
+    * @see <a href="https://sldn.softlayer.com/reference/services/softlayer_network_lbaas_member/addloadbalancermembers" />
+    */
+   @POST
+   @Named("LoadBalanceMember:addLoadBalancerMembers")
+   @Path("SoftLayer_Network_LBaaS_Member/addLoadBalancerMembers")
+   @QueryParams(keys = "objectMask", values = NAME_MASK)
+   @Produces(MediaType.APPLICATION_JSON)
+   @Fallback(NullOnNotFoundOr404.class)
+   LoadBalancer addLoadBalancerMembers(@WrapWith("parameters") List<Object> LoadBalancerMembersToBeAdded);
```

Comment:
is there any better type than `Object`?

Reply:
With a lot of the LBaaS APIs, they are expecting the payload to be in the format of the mixture of LB uuid and the definition. For example, for `updateLoadBalancerMembers`, the payload is something like `{"parameters": ["<load_balancer_uuid>", [{"uuid": "<load_balancer_member_uuid>", "weight": <weight>}]]}`.